spec/koans.js

```diff
@@ -301,102 +347,116 @@ describe('destructuring objects. ', () => {
+    const [a = 1] = [];
+    expect(a).toEqual(1) //ok //expect(a).toEqual(1)
   });
 
   it('for a missing value', () => {
-    const [a,b,c] = [1,,3];
-    //expect(b).toEqual(2);
+    const [a, b = 2, c] = [1, , 3];
+    expect(b).toEqual(2); //expect(b).toEqual(2);
   });
 
   it('in an object', () => {
-    const [a, b] = [{a: 1}];
-    //expect(b).toEqual(2);
+    const [a, b = 2] = [{
+      a: 1
+    }];
+    expect(b).toEqual(2); // ok //expect(b).toEqual(2);
   });
 
   it('if the value is undefined', () => {
-    const {a, b} = {a: 1, b: void 0};
-    //expect(b).toEqual(2);
+    const {
+      a,
+      b = 2
+    } = {
+      a: 1,
+      b: void 0
+    };
+    expect(b).toEqual(2); // ok //expect(b).toEqual(2);
   });
 
   it('also a string works with defaults', () => {
-    const [a, b] = '1';
-    //expect(a).toEqual('1');
-    // expect(b).toEqual(2);
+    const [a, b = 2] = '1';
+    expect(a).toEqual('1'); //ok //expect(a).toEqual('1');
+    expect(b).toEqual(2); //ok // expect(b).toEqual(2);
   });
 
 });
 
 /*********************************
-********* DAILY EXERCISE *********
-**********************************/
+ ********* DAILY EXERCISE *********
+ **********************************/
 
 describe('arrow functions. ', () => {
 
-  it('are shorter to write', function() {
+  it('are shorter to write', function () {
     let func = () => {
-      /*........*/
+      return 'I am func'; /*........*/
     };
-    // expect(func()).toBe('I am func');
+    expect(func()).toBe('I am func'); // ok // expect(func()).toBe('I am func');
   });
 
-  it('a single expression, without curly braces returns too', function() {
-    /*let func = () => .........;*/
-    //expect(func()).toBe('I return too');
+  it('a single expression, without curly braces returns too', function () {
+    let func = () => 'I return too'; //ok /*let func = () => .........;*/
+    expect(func()).toBe('I return too'); // ok //expect(func()).toBe('I return too');
   });
 
   it('one parameter can be written without parens', () => {
-   /* let func = ........;*/
-    //expect(func(25)).toEqual(24)
+    let func = (num) => --num //ok/* let func = ........;*/
+    expect(func(25)).toEqual(24); // ok//expect(func(25)).toEqual(24)
   });
 
   it('many params require parens', () => {
-    /* let func = ........;*/
-    //expect(func(23,42)).toEqual(23+42)
+    let func = (a, b) => a + b; //ok /* let func = ........;*/
+    expect(func(23, 42)).toEqual(23 + 42) // ok//expect(func(23,42)).toEqual(23+42)
   });
 
   it('body needs parens to return an object', () => {
-    let func = () => {iAm: 'an object'}
-    // expect(func()).toEqual({iAm: 'an object'});
+    let func = () => ({
+      iAm: 'an object'
+    })
+    expect(func()).toEqual({
+      iAm: 'an object'
+    }); //ok  // expect(func()).toEqual({iAm: 'an object'});
   });
 
   class LexicallyBound {
 
     getFunction() {
       return () => {
-        return new LexicallyBound(); /*changes might go here*/
```

Comment:
Para resolver los dos que no te salieron, había que poner `this` en donde los tests pudieran ver si es lexically bound. No es tan fácil de entender esta parte, pero en este caso en específico, `return new LexicallyBound()` no tiene sentido, ya que `LexicallyBound()` es la clase y se estaría creando a sí misma. En este punto, tenías que cambiarlo por `return this` para que el test tomara ese `this` y analizara su scope.